src/footer/Footer.js

```diff
@@ -12,32 +14,70 @@ const FooterContainer = styled.footer`
 const FooterContent = styled.div`
   align-items: center;
   display: flex;
-  justify-content: space-between;
-  margin: 0 auto;
-  max-width: ${CONTAINER_WIDTH}px;
+  flex-direction: column;
```

Comment:
The way this works is that you define your default and mobile device styles first and then you add/override styles by setting breakpoints at increasingly larger screen sizes